api/filesystem.go

```diff
@@ -121,6 +121,8 @@ func (fs *FileSystem) Upload(lpath, index string, toEncrypt bool) (string, error
 			var hash storage.Address
 			var wait func(context.Context) error
 			ctx := context.TODO()
+
+			// Fixing the pin counter as DONT_PIN , since this is deprecated
```

Comment:
comment?

Reply:
I changed this file previously and reverted it to previous state now.  So nothing changed.